docs/integration/infra.md

```diff
@@ -0,0 +1,276 @@
+title: Infrastructure Monitoring Integration
+description: Infrastructure performance monitoring - metrics, events, processes, packages, log collection and parsing for bare-metal servers, virtual machines and containers with support for Docker, containerd, k8s, swarm, nomad, ecs, eks, aks, gke.
+
+The Sematext Infra Integration uses [Sematext Agent](../agents/sematext-agent) to collect metrics, events, processes, packages for bare-metal servers, virtual machines and containers. Container monitoring is supported for Docker, containerd, k8s, Swarm, Nomad, ECS, EKS, AKS, GKE and more.
+
+We at Sematext aim to save you time and effort by giving you a strong starting point for monitoring your infrastrcture. You will **not** have to:
+
+- figure out which metrics to collect and which ones to ignore
+- give metrics meaningful labels
+- hunt for metric descriptions in the docs so that you know what each of them actually shows
+- build charts to group metrics that you really want on the same charts, not N separate charts
+- figure out, for each metric, which aggregation to use (min? max? avg? something else?)
```

Comment:
figure out, for each metric, which aggregation to use (min? max? avg? something else?)
=>
figure out the most useful aggregation to use for each metric (min? max? avg? something else?)